src/components/data_comps/dice/dice_comp_mod.F90

```diff
@@ -94,36 +97,11 @@ module dice_comp_mod
   !  real(R8)    , pointer :: ifrac0(:)
 
   !--------------------------------------------------------------------------
-  integer(IN),parameter :: ktrans = 42
-  character(16),parameter  :: avofld(1:ktrans) = &
-     (/"So_t            ","So_s            ","So_u            ","So_v            ", &
-       "So_dhdx         ","So_dhdy         ","Fioo_q          ","Sa_z            ", &
-       "Sa_u            ","Sa_v            ","Sa_ptem         ","Sa_tbot         ", &
-       "Sa_shum         ","Sa_dens         ","Faxa_swndr      ","Faxa_swvdr      ", &
-       "Faxa_swndf      ","Faxa_swvdf      ","Faxa_lwdn       ","Faxa_rain       ", &
-       "Faxa_snow       ","Si_t            ","Si_tref         ","Si_qref         ", &
-       "Si_ifrac        ","Si_avsdr        ","Si_anidr        ","Si_avsdf        ", &
-       "Si_anidf        ","Faii_taux       ","Faii_tauy       ","Faii_lat        ", &
-       "Faii_sen        ","Faii_lwup       ","Faii_evap       ","Faii_swnet      ", &
-       "Fioi_swpen      ","Fioi_melth      ","Fioi_meltw      ","Fioi_salt       ", &
-       "Fioi_taux       ","Fioi_tauy       " /)
-
-  character(16),parameter  :: avifld(1:ktrans) = &
-     (/"to              ","s               ","uo              ","vo              ", &
-       "dhdx            ","dhdy            ","q               ","z               ", &
-       "ua              ","va              ","ptem            ","tbot            ", &
-       "shum            ","dens            ","swndr           ","swvdr           ", &
-       "swndf           ","swvdf           ","lwdn            ","rain            ", &
-       "snow            ","t               ","tref            ","qref            ", &
-       "ifrac           ","avsdr           ","anidr           ","avsdf           ", &
-       "anidf           ","tauxa           ","tauya           ","lat             ", &
-       "sen             ","lwup            ","evap            ","swnet           ", &
-       "swpen           ","melth           ","meltw           ","salt            ", &
-       "tauxo           ","tauyo           " /)
+  integer(IN),parameter :: ktrans = 1
+  character(16),parameter  :: avofld(1:ktrans) = (/"Si_ifrac        "/)
+  character(16),parameter  :: avifld(1:ktrans) = (/"ifrac           "/)
```

Comment:
I'll trust you here, but I'm kind of surprised that ice fraction is the only field that dice is responsible for. I would have thought that it would be responsible for things like albedo (or SWup) and various fluxes. 

Does this have to do with the fact that we sometimes run CICE in prescribed mode - i.e., whenever we run with an active atmosphere, we also are using CICE rather than dice? Is this a fundamental characteristic of the system (i.e., of cime), or is it possible that someone would want to use cime with an active atmosphere and dice? If the latter, then should some of these fields be retained?

Reply:
Here is where the data models are a bit tricky. DICE does send albedos - but it does not get them from data files - it creates them explicitly - via code blocks like the following:
          i2x%rAttr(kavsdr,n) = ax_vsdr
          i2x%rAttr(kanidr,n) = ax_nidr
          i2x%rAttr(kavsdf,n) = ax_vsdf
          i2x%rAttr(kanidf,n) = ax_nidf
The key point is that the ONLY fields it reads from the data files are specified in namelist_definition_dice.xml and its ifrac. So it was very confusing to have all of those extra fields specified. The same holds for datm and dlnd - but I did not want to take to much on in this PR.

Reply:
I had the same question.  So item 3 is the new list of variables read from files?

Reply:
@rljacob - yes that is correct.